ui/src/components/Terminal/index.vue

```diff
@@ -102,6 +108,12 @@ onMounted(() => {
     setShareCode(props.shareCode);
   }
 
+  watchEffect(() => {
+    if (props.contentType && props.contentType === 'file-manager') {
+      sendEventToLuna(WINDOW_MESSAGE_TYPE.CREATE_FILE_CONNECT_TOKEN, '', lunaId.value, origin.value);
+    }
+  });
+
   initializeSocketEvent(terminalInstance, socket.value, t);
 });
 </script>
```

Comment:
The current code snippet does not appear to contain any discrepancies that require attention. However, potential areas of improvement can include optimizing performance by minifying the bundle using `terser`, checking dependencies for updates to ensure compatibility with latest versions used within the project, and ensuring proper handling of asynchronous operations during initialization phases.